cypress/integration/editorMode.spec.ts

```diff
@@ -7,25 +7,18 @@
 describe('Editor mode from URL parameter is used', () => {
   it('mode view', () => {
     cy.visitTestEditor('view')
-    cy.get('.splitter.left')
-      .should('have.class', 'd-none')
-    cy.get('.splitter.right')
-      .should('not.have.class', 'd-none')
+    cy.get('.splitter.left').should('have.class', 'd-none')
+    cy.get('.splitter.right').should('not.have.class', 'd-none')
```

Comment:
maybe replace this with "should not be visible"?

Reply:
done